UEBPrep/app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,15 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
```

Comment:
Cool, so I can delete all the session files without changing the functionality created this past week?